### PR TITLE
Prep for v1.20.1

### PR DESCRIPTION
**Pull request by odow**
## TODO

 - [x] #2285
 - [x] #2290

## Basic

 - [x] `version` field of `Project.toml` has been updated
       - If a breaking change, increment the MAJOR field and reset others to 0
       - If adding new features, increment the MINOR field and reset PATCH to 0
       - If adding bug fixes or documentation changes, increment the PATCH field

## Documentation

 - [x] Add a new entry to `docs/src/changelog.md`, following existing style

## Tests

 - [x] https://github.com/jump-dev/MathOptInterface.jl/actions/runs/6267756197

### CODE REVIEW

File: docs/src/changelog.md [hunk 1]
```diff
@@ -7,6 +7,26 @@ CurrentModule = MathOptInterface
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.20.1 (September 22, 2023)
+
+### Fixed
+
+ - Removed incorrect `kwargs` in some `copy_to` methods (#2272)
+ - Fixed [`ConstraintDualStart`](@ref) for [`Bridges.Constraint.SplitIntervalBridge`](@ref)
+   (#2275)
+ - Fixed `MethodError` when trying to modify a variable objective (#2278)
```
Comment (odow) [inline docs/src/changelog.md:17]:
```suggestion
 - Fixed `MethodError` when trying to modify a variable objective (#2278)
 - Fixed stack-overflow in `Utilities.operate(+,...)` with many arguments (#2285)
```

File: docs/src/changelog.md [hunk 2]
```diff
@@ -7,6 +7,27 @@ CurrentModule = MathOptInterface
...
+
+### Other
+
+ - Added MathOptSetDistances to `solver-tests.yml` (#2265)
+ - Updated Documenter (#2266)
+ - Fixed various JET errors (#2267) (#2269) (#2270) (#2271) (#2276) (#2277)
+ - Various style improvements
+   - Replaced `using Package` with `import Package` where possible (#2274)
+   - Removed `Utilities.EMPTYSTRING` (#2283)
+   - Removed unecessary `const` acronyms in `Utilities` (#2280) (#2281)
```
Comment (odow) [inline docs/src/changelog.md:28]:
```suggestion
   - Removed unnecessary `const` acronyms in `Utilities` (#2280) (#2281)
```

File: docs/src/changelog.md [hunk 3]
```diff
@@ -7,6 +7,27 @@ CurrentModule = MathOptInterface
...
+ - Fixed [`ConstraintDualStart`](@ref) for [`Bridges.Constraint.SplitIntervalBridge`](@ref)
+   (#2275)
+ - Fixed `MethodError` when trying to modify a variable objective (#2278)
+ - Fixed stack-overflow in `Utilities.operate(+,...)` with many arguments (#2285)
+
+### Other
+
+ - Added MathOptSetDistances to `solver-tests.yml` (#2265)
+ - Updated Documenter (#2266)
+ - Fixed various JET errors (#2267) (#2269) (#2270) (#2271) (#2276) (#2277)
```
Comment (odow) [inline docs/src/changelog.md:24]:
```suggestion
 - Fixed various JET errors (#2267) (#2269) (#2270) (#2271) (#2276) (#2277) (#2289)
```

**Review by blegat — CHANGES_REQUESTED** [review]
Let's hold off until my concern in https://github.com/jump-dev/MathOptInterface.jl/issues/2284 is addressed, I think we should revert https://github.com/jump-dev/MathOptInterface.jl/pull/2285 as it might cause more issues by fixing an invalid use case

File: docs/src/changelog.md [hunk 4]
```diff
@@ -7,6 +7,27 @@ CurrentModule = MathOptInterface
...
+### Other
+
+ - Added MathOptSetDistances to `solver-tests.yml` (#2265)
+ - Updated Documenter (#2266)
+ - Fixed various JET errors (#2267) (#2269) (#2270) (#2271) (#2276) (#2277) (#2289)
+ - Various style improvements
+   - Replaced `using Package` with `import Package` where possible (#2274)
+   - Removed `Utilities.EMPTYSTRING` (#2283)
+   - Removed unnecessary `const` acronyms in `Utilities` (#2280) (#2281)
+   - Removed invalid and unused method (#2286)
```
Comment (odow) [inline docs/src/changelog.md:29]:
```suggestion
   - Removed invalid and unused method (#2286)
 - Refactored `src/Utilities/model.jl` (#2287)
```

**Review by blegat — APPROVED** [review]
We can release, the benchmark of https://github.com/jump-dev/MathOptInterface.jl/issues/2284#issuecomment-1732136438 shows there is no regression

File: docs/src/changelog.md [hunk 5]
```diff
@@ -7,6 +7,28 @@ CurrentModule = MathOptInterface
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.20.1 (September 22, 2023)
```
Comment (odow) [inline docs/src/changelog.md:10]:
```suggestion
## v1.20.1 (September 24, 2023)
```

File: docs/src/changelog.md [hunk 6]
```diff
@@ -7,6 +7,28 @@ CurrentModule = MathOptInterface
...
+
+### Fixed
+
+ - Removed incorrect `kwargs` in some `copy_to` methods (#2272)
+ - Fixed [`ConstraintDualStart`](@ref) for [`Bridges.Constraint.SplitIntervalBridge`](@ref)
+   (#2275)
+ - Fixed `MethodError` when trying to modify a variable objective (#2278)
+ - Fixed stack-overflow in `Utilities.operate(+,...)` with many arguments (#2285)
```
Comment (odow) [inline docs/src/changelog.md:18]:
```suggestion
 - Fixed stack-overflow in `Utilities.operate(+,...)` with many arguments (#2285) (#2290)
```

**Pull request merged**